packages/plugin-rsc/src/plugin.ts

```diff
@@ -1267,14 +1267,16 @@ function vitePluginUseClient(
               const meta = manager.clientReferenceMetaMap[id]
               if (meta) {
                 meta.renderedExports = mod.renderedExports
+                const normalized = normalizePath(
+                  path.relative(
+                    manager.config.root,
+                    chunk.facadeModuleId ?? [...chunk.moduleIds].sort()[0]!,
```

Comment:
TODO: probably we should find a better heuristics than `[...chunk.moduleIds].sort()[0]`